src/auth/basic/LDAP/basic_ldap_auth.cc

```diff
@@ -23,7 +23,7 @@
  *
  * Usage: squid_ldap_auth -b basedn [-s searchscope]
  *                        [-f searchfilter] [-D binddn -w bindpasswd]
- *                        [-u attr] [-h host] [-p port] [-P] [-R] [ldap_server_name[:port]] ...
+ *                        [-u attr] [-h host] [-p port] [-B] [-P] [-R] [ldap_server_name[:port]] ...
```

Comment:
Since Squid-3.x are no longer supported this option can be removed from this merge PR. (People back-porting may want it, but not the official merge).